packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -216,6 +216,9 @@ const ReviewVoteTableRow = ({ post, dispatch, costTotal, classes, expandedPostId
   // note: this needs to be ||, not ??, because quadraticScore defaults to 0 rather than null
   const userReviewVote = post.currentUserReviewVote?.quadraticScore || qualitativeScoreDisplay;
 
+  const visitedDate = markedVisitedAt || post.lastVisitedAt
+  const unreadComments = visitedDate ? visitedDate < post.lastCommentedAt : true
```

Comment:
I think numeric comparisons with null always return `false`, so this might actually be changing existing behavior (since now a `visitedDate` of `null` will cause `unreadComments` to be `true` rather than `false`).  But I'd double-check that.  (It seems plausible to me that the new behavior would be correct, but want to make sure it's intended.)

Reply:
Checked in browser console. `null < 5` evaluates to `true`
<img width="100" alt="image" src="https://github.com/ForumMagnum/ForumMagnum/assets/12041390/1b91f5d5-729d-4e7c-bca2-5afa5d9b5c7b">

Reply:
Oh, I guess this is only for `undefined`?  (I'd still prefer to not be accidentally doing numeric comparisons with `null`, I think)